pom.xml

```diff
@@ -129,7 +129,51 @@
               <exclude>src/pl/**/*</exclude>
               <exclude>src/port/*</exclude>
               <exclude>src/template/*</exclude>
-              <exclude>src/test/**/*</exclude>
+              <exclude>src/test/**/*.c</exclude>
```

Comment:
I'm wondering whether we should not exclude some of the text files (source files, makefiles, etc) which we could add header files.  I remember we have a auto-script for header adding. @linwen